src/app/containers/Video/index.jsx

```diff
@@ -43,16 +47,31 @@ const VideoContainer = ({ blocks }) => {
     ['blocks', 1, 'model', 'blocks', 0, 'model', 'locator'],
     aresMediaBlock.model,
   );
-  const items = [
-    {
-      versionID,
-      kind,
-      duration,
+  const guidance = deepGet(['warnings', 'short'], version);
+  const id = `mp#${pid}`;
+  const mediaPlayerSettings = {
+    product: 'news',
+    responsive: true,
+    statsObject: { clipPID: pid },
+    mediator: {
+      host: mediatorURL(env),
+    },
+    playlistObject: {
+      title,
+      holdingImageURL: `https://${holdingImageUrl}`,
+      guidance,
+      items: [
+        {
+          versionID,
+          duration,
+          kind,
+        },
+      ],
     },
```

Comment:
To address the subtitle issue raised by James, it should be sufficient to add these properties to the `mediaPlayerSettings` object:

```
ui: {
    subtitles: {
      defaultOn: false,
    },
    locale: {
      lang: 'en-GB',
    },
  },
```